hack/prepare-irso-tests.sh

```diff
@@ -9,12 +9,35 @@ CLUSTER_TYPE="${CLUSTER_TYPE:-kind}"
 CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-podman}"
 IRONIC_CUSTOM_IMAGE=${IRONIC_CUSTOM_IMAGE:-localhost/ironic:test}
 
-"${CONTAINER_RUNTIME}" build -t "${IRONIC_CUSTOM_IMAGE}" .
+build_args=()
```

Comment:
```suggestion
declare -a build_args=()
```

Reply:
It is just standard good practice of writing cleaner, explicit code. `declare -a` is not mandatory unlike `declare -A` for associative arrays, but if we check the google style guide for example (which we have agreed to loosely follow), declaration is always there.

I could also ask a counter-question, why not declare it? :)

Reply:
For me it's just redundant information, but I'm fine with changing it.